protocol/avdl/keybase1/common.avdl

```diff
@@ -328,6 +328,7 @@ protocol Common {
   record FullNamePackage {
     FullNamePackageVersion version;
     FullName fullName;
+    string bio;
```

Comment:
will it still be able to use existing on-disk cache? It should just load FullNamePackages with `bio` being empty, right?